account_cash_invoice/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2017 Creu Blanca
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    'name': 'Account cash invoice',
+    'version': '12.0.1.0.0',
+    'category': 'Accounting',
+    'author': "Creu Blanca,"
```

Comment:
Non blocking, but I think it would be a little nicer if you use single quote to all. :)